google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -47,8 +47,9 @@ appengine.cloudsdk.java.components.howtoinstall=To install, run "gcloud componen
 appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
 appengine.cloudsdk.location.badchars.message=The specified Cloud SDK directory contains invalid characters or starts or ends with spaces.
-appengine.cloudsdk.location.invalid.message=No Cloud SDK was found in this directory.
+appengine.cloudsdk.location.invalid.message=No Cloud SDK was found in the specified directory.
 appengine.cloudsdk.location.missing.message=Cloud SDK home directory is not specified.
+appengine.cloudsdk.location.missing.message.deployui=The Cloud SDK is misconfigured. To fix, reconfigure the Server above.
```

Comment:
why is location.missing.message 'not specified', but ....message.deployui 'misconfigured'? Can't we just reuse appengine.cloudsdk.location.invalid.message for this error condition?